### PR TITLE
FUSETOOLS-2074 - update 2.1.x for latest bpmn2 modeler

### PR DESCRIPTION
**Pull request by pleacu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```
Comment (apupier) [inline pom.xml]: shouldn't we create a separate commit/PR for this line update?

Comment (bfitzpat) [inline pom.xml]: It could be done in a separate PR, but like the changes for the BPMN2 modeler these properties don't change a whole heck of a lot. If we document it in the JIRA is that enough? Seems like a lot of overkill to do a whole separate PR

**Comment by bfitzpat** [comment]
I'm good with a +1 if we update the JIRA about the line Aurelien noted since it really doesn't fit in with what the JIRA is supposed to be aimed at.

**Comment by pleacu** [comment]
I've updated the Jira problem description to include updating the dependent Fuse Tooling version.

File: pom.xml (hunk 1 above)
Comment (bfitzpat) [inline pom.xml]: @pleacu So the growing consensus with @lhein and @apupier is to split out the "fuse-tooling-url" change into a separate JIRA-driven PR. Can we have you update this PR to remove that change and put it into a separate PR with a new JIRA?

Comment (pleacu) [inline pom.xml]: My pleasure... :) - PR updated.  We might as well wait on the Fuse Tooling dependency since that'll change once
   it goes .Final.

    --paull



----- Original Message -----

**Comment by bfitzpat** [comment]
Thanks @pleacu. @apupier does that get a +1 from you?

@pleacu Can you create a JIRA for the dependent fuse tooling version update so we don't lose that?

File: pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta4/</fuse-tooling-url>
+    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```
Comment (lhein) [inline pom.xml]: We do a downgrade?

Comment (bfitzpat) [inline pom.xml]: Returned it to the way it was before - we'll update it when we get the final release URL. This is just a temporary

Comment (pleacu) [inline pom.xml]: Yes.  This was the part of my original PR that needs to be a separate Jira.  I suggested waiting
   until Fuse Tooling goes .Final before changing it.

    --paull



----- Original Message -----

**Comment by apupier** [comment]
+1
